staff/views/repos.py

```diff
@@ -100,7 +100,7 @@ def enhance(repo):
             return repo | {
                 "first_run": first_run,
                 "has_jobs": has_jobs,
-                "has_releases": "github-releases" in repo["topics"],
+                "has_outputs": "github-releases" in repo["topics"],
```

Comment:
I think `has_outputs` is not specific enough

I think it should explicitly mention github, so maybe `has_github_outputs`?